src/suresh/maxLength.java

```diff
@@ -0,0 +1,41 @@
+package suresh;
+
+public class maxLength {
+
+	String[] arr = { "Harsh,", "Maulik", "Technocredits", "krisna" };
+
+	static String getmaxLength(String[] arr) {
+		int maxLength = 0;
+		String longestString = null;
+
+		for (String s : arr)
+
+			if (s.length() > maxLength) {
```

Comment:
remove variable maxLength. you can use longestString.length().

if(s.length() > longestString.length())